docs/tools/vdb_table/data/activeloop.json

```
@@ -58,14 +58,14 @@
     "comment": "no native sparse vector support, although it supports all numpy arrays hence can also store sparse numpy arrays"
   },
   "bm25": {
-    "support": "none",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://docs.deeplake.ai/latest/",
```

I suggest linking to the subheader that has key features, so the reader knows immediately where to look. Anywhere else where the FTS functionality is more deeply documented would be more helpful than just a high level page.
```suggestion
    "source_url": "https://docs.deeplake.ai/latest/#key-features",
```